ffi/mupdf_h.lua

```diff
@@ -50,111 +50,102 @@ fz_matrix *fz_translate(fz_matrix *, float, float);
 fz_matrix *fz_pre_translate(fz_matrix *, float, float);
 fz_rect *fz_transform_rect(fz_rect *restrict, const fz_matrix *restrict);
 typedef struct fz_context_s fz_context;
+typedef struct fz_font_s fz_font;
+typedef struct fz_hash_table_s fz_hash_table;
 typedef struct fz_storable_s fz_storable;
+typedef struct fz_key_storable_s fz_key_storable;
 typedef void fz_store_drop_fn(fz_context *, fz_storable *);
 struct fz_storable_s {
   int refs;
   fz_store_drop_fn *drop;
 };
+struct fz_key_storable_s {
+  fz_storable storable;
```

Comment:
Four-space indentation, right? :-)

(I know, old file.)

Reply:
That's generated code by ffi-cdecl. :)